[WIP] Add tutorial (#1262)
collective/icalendar: docs/how-to/getting-started.rst

.. code-block:: py

    from icalendar import Event
    import datetime as dt
    import zoneinfo

    event = Event.new(
        start=dt.datetime(2026, 3, 21, 6, 30,0, tzinfo=zoneinfo.ZoneInfo('Europe/Berlin')),
        end=dt.datetime(2026, 3, 21, 7, 30,0, tzinfo=zoneinfo.ZoneInfo('Europe/Berlin')),
    )
    print(event.to_ical().decode())

This creates an event with a start and end time defined using ``datetime`` with a timezone. The output shows your new event:

.. code-block::

    BEGIN:VEVENT
    DTSTART;TZID=Europe/Berlin:20260321T063000
    DTEND;TZID=Europe/Berlin:20260321T073000
    DTSTAMP:20260301T104000Z
    UID:a3d2479f-7127-48e6-bbca-d9e0637f02b7
    END:VEVENT

Notice that the event component includes the start and end dates with their times (``DTSTART`` and ``DTEND``) along with a unique identifier (``UID``), and information about when this event was created (``DTSTAMP``).

Similarly to the calendar event you can edit these properties or add new ones.

.. code-block:: py

>     event.add('summary', 'Pickup bicycle from the workshop.')

Reviewer: ```suggestion
    event.add("summary", "Pickup bicycle from the workshop.")
```

> Using the :meth:`~icalendar.cal.component.Component.add()` method on a component you can add new properties with their name and value. The summary property represents the title of an event.

Reviewer: ```suggestion
Using the :meth:`~icalendar.cal.component.Component.add()` method on a component, you can add new properties with their name and value.
The summary property represents the title of an event.
```

> To use an event you must add it to the calendar.

Reviewer: ```suggestion
To use an event, you must add it to the calendar.
```
